Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
       rest-client (>= 1.6.7)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
-    mime-types (3.3)
+    method_source (1.0.0)
+    mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.3)
-    mini_magick (4.9.5)
+    mimemagic (0.3.4)
+    mini_magick (4.10.1)
```

Comment:
### [mini_magick](https://github.com/minimagick/minimagick)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.9.5 → 4.10.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/minimagick/minimagick/compare/v4.9.5...v4.10.1).

These are the individual commits:
- (db6eda3) [update rubies list; sort by newness (#479)](https://github.com/minimagick/minimagick/commit/db6eda3762a23727eb56dbeefa5c71bd8f518120)
- (916b895) [Prioritizes discovery of ImageMagick7/GraphicsMagick over le…](https://github.com/minimagick/minimagick/commit/916b89589a4486ba6d1d417b495a2affe0d08a7a)
- (259105c) [Implement #landscape? and #portrait? Image class methods to …](https://github.com/minimagick/minimagick/commit/259105c42790910939a570ffd6a66d56a0ff7b11)
- (bca935c) [Add Ruby 2.7 to TravisCI build matrix (#495)](https://github.com/minimagick/minimagick/commit/bca935cddc88b0f32dce6fb4aef50e9e90392382)
- (1b3fe59) [Fix Ruby 2.7 warnings (#496)](https://github.com/minimagick/minimagick/commit/1b3fe5928aeb92a4ddade9f89db4ba610ebbef82)
- (adb8283) [Bump to 4.10.0](https://github.com/minimagick/minimagick/commit/adb8283194815a8f65b731d934293ba3d4b85a8c)
- (4749aab) [Still prioritize ImageMagick before GraphicsMagick](https://github.com/minimagick/minimagick/commit/4749aabd381b5a14c19aa5afc37a4c1de505559c)
- (4206a74) [Bump to 4.10.1](https://github.com/minimagick/minimagick/commit/4206a74d374a84a91e429628045e44559e042dc4)


</details>